MVP Module/Source/Feed Presentation/FeedPresenter.swift

```
@@ -13,25 +13,34 @@ protocol FeedView {
 	func display(_ viewModel: FeedViewModel)
 }
 
+protocol FeedErrorView {
+	func display(_ viewModel: FeedErrorViewModel)
+}
+
 final class FeedPresenter {
 	private let feedView: FeedView
 	private let loadingView: FeedLoadingView
+	private let feedErrorView: FeedErrorView
 
-	init(feedView: FeedView, loadingView: FeedLoadingView) {
+	init(feedView: FeedView, loadingView: FeedLoadingView, feedErrorView: FeedErrorView) {
 		self.feedView = feedView
 		self.loadingView = loadingView
+		self.feedErrorView = feedErrorView
 	}
 
 	func didStartLoadingFeed() {
+		feedErrorView.display(FeedErrorViewModel(isError: false))
 		loadingView.display(FeedLoadingViewModel(isLoading: true))
 	}
 
 	func didFinishLoadingFeed(with feed: [FeedImage]) {
+		feedErrorView.display(FeedErrorViewModel(isError: false))
```

Since the error is hidden on `didStartLoadingFeed`, we don't need to hide it again in this `didFinishLoadingFeed(with feed:)` method. You can remove this line and the tests still pass:

```suggestion
```